.github/workflows/ci.yml

```diff
@@ -43,6 +43,29 @@ jobs:
     - run: chmod +x keyboard-configurator-x86_64.AppImage
     - run: xvfb-run ./keyboard-configurator-x86_64.AppImage --help-gtk
 
+  linux-arm64:
+    runs-on: ubuntu-20.04
+    steps:
+    - run: echo "Running on ARM64!"
+    - run: sudo apt-get update && sudo apt-get install cargo libgtk-3-dev libhidapi-dev libudev-dev patchelf
+    - run: cd linux && ./build.py --arm64 $RELEASE
```

Comment:
This is missing a `uses: actions/checkout@v4` step to first checkout the repository.

Reply:
Added back.